docs/running-your-own-experiment/running_an_experiment.md

```diff
@@ -56,6 +56,9 @@ trials: 5
 # 1 day = 24 * 60 * 60 = 86400
 max_total_time: 86400
 
+# The docker registry for your fuzzbench experiments.
```

Comment:
Make the example `gcr.io/$FUZZBENCH` and mention that this is the default if using Google cloud.